client/src/components/main/chatwindow/CreateComment.js

```diff
@@ -0,0 +1,87 @@
+import React, { useState} from 'react';
+import styled from 'styled-components';
+import axios from 'axios';
+
+/*
+Posts new comment to database when enter key pressed
+Set newComment to true using Parent's dispatch
+*/
+
+const jwt = '';
+const config = { headers: { authorization: `bearer ${jwt}` } };
+
+const CreateComment = props => {
+  const { setNewComment } = props;
+  const [comment, setComment] = useState('');
+
+  const handleSubmit = text => {
+    axios
+      .post('/api/comments', { user: '5e4deef29b70942e38591863', text: text }, config)
+      .catch(err => console.error(err));
+
+    setComment('');
+  };
+
+  const handleEnter = event => {
```

Comment:
What's the difference between `handleEnter` and `handleSubmit`?

Reply:
`handleEnter` is an event handler that calls `handleSubmit` when the enter key is pressed.

`handleSubmit` is the event handler that processes the submit event on the create comment form. The submit event is posting the comment to the db.

So, together, when enter is pressed within the textarea input, submit is triggered.

Reply:
Could we have enter directly trigger `onSubmit`?